openhcl/virt_mshv_vtl/src/processor/tdx/mod.rs

```diff
@@ -1097,6 +1101,11 @@ impl UhProcessor<'_, TdxBacked> {
     }
 
     async fn run_vp_tdx(&mut self, dev: &impl CpuIo) -> Result<(), VpHaltReason<UhRunVpError>> {
+        let next_vtl = self
```

Comment:
Update all the bits in here to use next_vtl?

Reply:
I didn't yet because I wanted to be able to go through and more thoughtfully do it, make sure I look at the full fallout.